aggregate_spatial.json

```
@@ -0,0 +1,130 @@
+{
+    "id": "aggregate_spatial",
+    "summary": "Compute zonal statistics for a vector data cube",
+    "description": "Aggregates zonal statistics for features within a vector data cube (e.g., one or multiple points or polygons) over the spatial dimensions.\n\nIn case of polygon features, the process considers all pixels for which the point at the pixel center intersects with the corresponding polygon (as defined in the Simple Features standard by the OGC).\n\nThe data cube must have been reduced to only contain two raster dimensions and a third dimension the values are aggregated for, for example the temporal dimension to get a time series. Otherwise this process fails with the `TooManyDimensions` error.\n\nThe number of total and valid pixels is returned together with the calculated values.",
+    "categories": [
+        "cubes",
+        "aggregate & resample"
+    ],
+    "experimental": true,
+    "parameter_order": [
+        "data",
+        "polygons",
```

```suggestion
        "geometries",
```
or `vector` to fix the build.